CONTRIBUTING.md

```diff
@@ -7,18 +7,61 @@ instructions below to make your ideas come to life faster!
 
 ### Prepare
 
-1. Install [nodejs][nodejs] and [yarn][yarn].
-2. Install [Visual Studio Code][vscode] with the
+Here's what you need to do before you start working on the repo.
+
+Steps for contributors:
+
+1. If you don't have a GitHub account, [create one][createaccount].
+2. Install [Git][git], [Node.js][nodejs] and [Yarn][yarn]. These are essential tools for
```

Comment:
Try to fit in the width of 80 characters in Markdown.

Reply:
fixed in https://github.com/AdguardTeam/AdguardFilters/pull/149292/commits/fa6cc78ad2e8d009256d7a9d8a5143d1feecac35